projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_input-group.scss

```diff
@@ -54,3 +53,37 @@ $_dark-bootstrap-input-group: extend(
     $_bootstrap-input-group,
     $_base-dark-input-group
 );
+
+/// Generates an dark indigo input-group schema
+/// @type {Map}
+/// @prop {map} success-secondary-color [igx-color: ('success'), rgba: .3, hexrgba: #fff] - The success color used in the valid state.
+/// @prop {map} warning-secondary-color [igx-color: ('warn'), rgba: .3, hexrgba: #fff] - The warning color used in the warning state.
+/// @prop {map} error-secondary-color [igx-color: ('error'), rgba: .3, hexrgba: #fff] - The error color used in the error state.
```

Comment:
The alpha channel in all three should be `.7` according to the implementation below.
Also, I feel it would be better if hexrgba/to-opaque used the surface color as the base.